app/src/common/shared/com/igalia/wolvic/VRBrowserActivity.java

```diff
@@ -677,6 +677,7 @@ protected void onNewIntent(final Intent intent) {
     public void onConfigurationChanged(Configuration newConfig) {
         Language language = LocaleUtils.getDisplayLanguage(this);
         newConfig.setLocale(language.getLocale());
+        // TODO: updateConfiguration(Configuration,DisplayMetrics) in Resources has been deprecated
```

Comment:
Let's create a new issue for this please and link it here.

Reply:
https://github.com/Igalia/wolvic/issues/797